qt/python/mantidqt/mantidqt/dialogs/errorreports/test/test_errorreport_presenter.py

```diff
@@ -33,7 +33,7 @@ def setUp(self):
             self.view, self.exit_code, application=self.app_name)
         self.view.CONTACT_INFO = "ContactInfo"
         self.view.NAME = 'John Smith'
-        self.view.EMAIL = 'john.smith"email.com'
+        self.view.EMAIL = 'john.smith@email.com'
```

Comment:
Nit can this be `@example.com`
The RFC for emails asks users to use `example.com` since other domains are probably owned by someone who then has to process these emails